src/css/graph.css

```diff
@@ -110,6 +176,7 @@
 }
 
 .scatter-line.hovering, .scatter-point.hovering {
+  cursor: pointer;
```

Comment:
I'm not sure if this is semantically correct, since it's not something people are clicking on. Let's ask Rong.